Cargo.toml

```diff
@@ -16,14 +16,17 @@ byteorder = "1.4.3"
 crc = "3.0.0"
 log = { version = "0.4.17", optional = true }
 env_logger = { version = "0.9.0", optional = true }
+core2 = "0.4"
 
 [dev-dependencies]
 rust-lzma = "0.5"
 
 [features]
+default = ["std"]
 enable_logging = ["env_logger", "log"]
 stream = []
 raw_decoder = []
+std = []
```

Comment:
In terms of features, "no_std" should be an opt in, as it pulls an additional crate (core2). This crate should be optional, and only pulled for users who explicitly opt into the "no_std" feature.

Reply:
I can change this. Makes sense to do opt-in for pulling in optional dependency  crates.